code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -378,6 +380,45 @@
 	anchored = FALSE
 	target = null
 
+
+/mob/living/simple_animal/bot/cleanbot/larry/attackby(obj/item/I, mob/living/user)
+	if(I) //Does the arg exist?
+		if(istype(I, /obj/item/kitchen/knife)) //Is it a knife?
+			if(!knife)
```

Comment:
```suggestion
	if(istype(I, /obj/item/kitchen/knife))
```
bruh

Reply:
null cannot be knife

Reply:
Logic chain for this completely changed.